docs/reference/search/search-your-data.asciidoc

```diff
@@ -227,9 +177,45 @@ GET /*/_search
 ----
 // TEST[setup:my_index]
 
-include::request/index-boost.asciidoc[]
-include::request/preference.asciidoc[]
-include::request/search-type.asciidoc[]
+[discrete]
+[[common-search-options]]
+=== Common search options
+
+You can use the following options to customize your searches.
+
+*Paginate search results* +
+By default, searches return only the top 10 matching hits. To retrieve
+more or fewer documents, see <<paginate-search-results>>.
+
+*Retrieve selected fields* +
+The search response's `hit.hits` property includes the full document
+<<mapping-source-field,`_source`>> for each hit. To retrieve only a subset of
+the `_source` or other fields, see <<search-fields>>.
+
+*Sort search results* +
+By default, search hits are sorted by `_score`, a <<relevance-scores,relevance
+score>> that measures how well each document matches the query. To customize the
+calculation of these scores, use the
+<<query-dsl-function-score-query,`function_score`>> or
```

Comment:
We can avoid emphasizing `function_score` since it's deprecated, and we're trying to replace it with `script_score`.